package.json

```diff
@@ -40,7 +40,8 @@
         "@types/node": "^18.6.5",
         "@typescript-eslint/eslint-plugin": "^5.4.0",
         "@typescript-eslint/parser": "^5.4.0",
-        "chai": "^4.3.4",
+        "chai": "^4.3.6",
+        "chai-as-promised": "^7.1.1",
```

Comment:
Try not to introduce new dependencies, chai is only used as an `assert` polyfill for browser tests and we don't use "expect" style testing